lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -279,7 +279,7 @@ end
 
 function M.make_directory_change_for_command(dir, command)
   if fn.has "win32" == 1 then
-    return string.format("pushd %s & %s & popd", cmdpath(dir), command)
+    return string.format([[cmd /C "pushd %s & %s & popd"]], dir, command)
```

Comment:
but will this also change the cwd for powershell. My idea was actually to execute everything `iter_cmd_sync` with cmd